.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,27 @@
+name: 🐛 Bug Report

```

Comment:
This caterpillar is easily confused with a snake. What do you think about using 🪲 (U+1FAB2) instead?

Reply:
Unfortunately I only see a gray rectangular for the proposed alternative on my mobile.

I had also this idea initially, than I realized that  🐛, U+1F41B is called Bug.

Reply:
:bug: was added in Emoji 1.0 while :bettle: was added in 13.0 (2020)... I still prefer :beetle: